applications/teletype/src/teletype_port_utils.erl

```diff
@@ -110,7 +110,10 @@ fix_port_request_data(JObj, DataJObj) ->
 
 -spec fix_numbers(kz_json:object(), kz_json:object()) -> kz_json:object().
 fix_numbers(JObj, _DataJObj) ->
-    NumbersJObj = kz_json:get_value(<<"numbers">>, JObj, kz_json:new()),
+    NumbersJObj = kz_json:get_json_value(<<"numbers">>
```

Comment:
@icehess its possible for there to be both numbers and ported_numbers on the document...